README.md

```diff
@@ -4,7 +4,7 @@
 
 Tikui is a MIT-licensed free software project allowing you to create a web pattern library.
```

Comment:
URL of public website tikui.org should be more highlighted in the README

Reply:
Is it better on the last version ?

Reply:
perfect :ok_hand: